policy/datasets.yaml

```diff
@@ -551,6 +556,13 @@ source:
     python: lsst.pipe.tasks.parquetTable.ParquetTable
     tables: raw
     template: ''
+fakes_source:
```

Comment:
Move down to fakes section, so all "src" datasets are together and all "fakes" datasets are together.